pkg/config/remote/service/service.go

```diff
@@ -216,7 +216,7 @@ func NewService(cfg model.Reader, apiKey, baseRawURL, hostname string, tags []st
 		return nil, err
 	}
 
-	dbPath := path.Join(cfg.GetString("run_path"), "remote-config.db")
+	dbPath := path.Join(cfg.GetString("run_path"), dbName)
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 Nitpick:** ‏We may want to sanitize `dbName` here (using `filepath.Clean` if `path.Join` doesn't do it), just in case

Reply:
`path.Join` does indeed `Clean()` the result before returning it 😁